components/build/shared-resources/disable-cso-cvo.yaml

```diff
@@ -0,0 +1,78 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scale-down-cso-cvo
+rules:
+  - apiGroups:
+      - apps
+    resources:
+      - deployments/scale
+    verbs:
+      - patch
+  - apiGroups:
+      - apps
+    resources:
+      - deployments
+    verbs:
+      - get
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: scale-down-cso-cvo
+  namespace: build-templates
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: scale-down-cso
+  namespace: openshift-cluster-storage-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: scale-down-cso-cvo
+subjects:
+  - kind: ServiceAccount
+    name: scale-down-cso-cvo
+    namespace: build-templates
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: scale-down-cvo
+  namespace: openshift-cluster-version
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: scale-down-cso-cvo
+subjects:
+  - kind: ServiceAccount
+    name: scale-down-cso-cvo
+    namespace: build-templates
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: scale-down-cso-cvo
+  namespace: build-templates
+  annotations:
+    argocd.argoproj.io/hook: PostSync
+spec:
+  template:
+    spec:
+      containers:
+        - image: registry.redhat.io/openshift4/ose-cli:v4.9
+          command:
+            - /bin/bash
+            - "-ec"
+            - |
+              oc scale --replicas=0 deploy/cluster-version-operator -n openshift-cluster-version
```

Comment:
while we need this and disabling the CSO below if 
a) tech preview is enabled
b) we want openshift builds to leverage shared resources

I am pretty sure we won't need disable-cso-cvo if we want to run shared resources in 
1) a non tech preview cluster
2) use them for pods or tekton resources

I'll bring up a cluster today, confirm/deny that, and report back @Michkov 

thanks

Reply:
That would be great since currently tests are failing because of disabled CSO

Reply:
interesting I see https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_infra-deployments/330/pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-deployment/1521821036058775552 and it says it is passing

was their some specific output within that job you are referring to @Michkov ?  For my own education as much as anything.

Regardless though, we still want to avoid disabling these two.

Reply:
hmm, previous run failed with `{Operator unavailable (ClusterServiceVersionNotSucceeded)`, so I did not expect that retest will fix that, maybe race condition.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_infra-deployments/330/pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-deployment/1521805910018625536

Reply:
Do not touching cso/cvo anymore